docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -112,7 +126,70 @@ step('Load the demo page', () => {
   await page.goto('https://elastic.github.io/synthetics-demo/'); <1>
 });
 ----
-<1> See the https://playwright.dev/docs/api/class-page#page-goto[`page.goto` reference] for more information.
+<1> Go to the https://playwright.dev/docs/api/class-page#page-goto[`page.goto` reference] for more information.
+
+[discrete]
+[[synthetics-step-ref]]
+== Arguments
+
+[cols="1,2"]
+|===
+| *`name`* (_string_)
+| A user-defined string to describe the journey.
+
+| *`callback`* (_function_)
+| A function where you simulate user workflows using Synthetics and <<synthetics-playwright,Playwright>> syntax.
+|===
+
+[[synthetics-create-test-script-recorder]]
+[NOTE]
+====
+If you want to generate code by interacting with a web page directly, you can use the *Synthetics Recorder*.
+
+The recorder launches a https://www.chromium.org/Home/[Chromium browser] that will listen to each interaction you have with the web page and record them internally using Playwright.
+When you're done interacting with the browser, the recorder converts the recorded actions into JavaScript code that you can use with Elastic Synthetics or {heartbeat}.
+
+For more details on getting started with the Synthetics Recorder, refer to <<synthetics-recorder>>.
+====
+
+[discrete]
+[[synthetics-playwright]]
+== Playwright syntax
+
+Inside the callback for each step, you'll likely use a lot of Playwright syntax.
+Many Playwright classes are supported in the Elastic Synthetics library to simulate
+user workflows including tasks like:
+
+* Interacting with the https://playwright.dev/docs/api/class-browser[browser]
+  or the current https://playwright.dev/docs/api/class-page[page] (like in the example above).
+* Finding elements on a web page using https://playwright.dev/docs/api/class-locator[locators].
+* Simulating https://playwright.dev/docs/api/class-mouse[mouse],
+  https://playwright.dev/docs/api/class-touchscreen[touch], or
+  https://playwright.dev/docs/api/class-keyboard[keyboard] events.
+
+Visit the https://playwright.dev/docs[Playwright documentation] for information.
+
+[NOTE]
+====
+Playwright functionality that is not supported in Elastic Synthetics includes:
```

Comment:
We might need to mention it as `Playwright functionality that is not supported out of the box`, as users can still access these by using different means via Synthetics ex: video and API request.